### ISSUE TITLE
stylus website underselling itself?

### ISSUE THREAD

**Issue by coderofsalvation** [stylus/stylus-lang.com#22]
Hi just wanted to give you some feedback from the outside world :)

I came across stylus multiple times now, while searching for a portable less/scss-replacement.
However, it wasn't clear to me that stylus does accept brackets (scss/less syntax) at all!

Therefore, i think many devs skim thru the site and will probably think 'oh this is kindof coffeescript for css..nevermind'.

I would consider the bracket/semicolon-omission a feature, rather than the USP.

My advice: communicate 'a portable,flexible,etc less/scss-replacement' in the header. That will probably increase adoption for curious less/scss developers.

**Comment by iChenLei** [comment]
Yes, you are right. I also think Stylus is kind of coffeescript for css when I am not Stylus maintainer.
